### ISSUE TITLE
run `wasm-pack build` failed when use web-sys on windows

### ISSUE THREAD

**Issue by huangjj27** [rustwasm/wasm-pack#399]
## 🐛 Bug description
import web-sys, and do nothing, then `wasm-pack build` complains:
```
Compilation of your program failed. stderr:

   Compiling web-sys v0.3.1
# file name, dir name or volume name syntax incorrect
error: couldn't read "\\\\?\\C:\\Users\\34937\\Desktop\\learning-rust\\wasm-pack-learn\\target\\wasm32-unknown-unknown\\release\\build\\web-sys-b58ff1058a8dc310\\out/bindings.rs": 文件名、目录名或卷标语法不正确。 (os error 123)    
  --> C:\Users\34937\.cargo\registry\src\mirrors.ustc.edu.cn-61ef6e0cd06fb9b8\web-sys-0.3.1\src\lib.rs:30:1
   |
30 | include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `web-sys`.

To learn more, run the command again with --verbose.
```
while the crate can be build just by running `cargo build --target=wasm32-unknown-unknown`.

#### 🤔 Expected Behavior
It should build too with `wasm-pack build`

#### 👟 Steps to reproduce
1. add edition & dependencies to Cargo.toml:
```
[package]
edition = "2018"

[dependencies]
web-sys = "0.3"
```
2. `use web-sys` in `$MY_CRATE/src/lib.rs`
3. run `wasm-pack build`. it would fail.
#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: 0.5.0
rustc version: rustc 1.30.0-nightly (bb0896af1 2018-09-29)

**Comment by xmclark** [comment]
Thank you @huangjj27! I believe this may be related to #380.

**Comment by alexcrichton** [comment]
This may also have been a [bug in wasm-bindgen](https://github.com/rustwasm/wasm-bindgen/issues/943) which was [since fixed](https://github.com/rustwasm/wasm-bindgen/pull/944) and should now be published as web-sys 0.3.2, @huangjj27 mind trying out the latest version of `web-sys` to see if it's fixed?

**Comment by huangjj27** [comment]
@alexcrichton it fix! I `cargo check` and pass! thanks!
